modules/shortcodes/dailymotion.php

```diff
@@ -327,3 +327,61 @@ function jetpack_dailymotion_embed_reversal( $content ) {
 }
 
 add_filter( 'pre_kses', 'jetpack_dailymotion_embed_reversal' );
+
+/**
+ * DailyMotion Widget Shortcode
+ *
+ * Example:
+ * [dailymotion-widget id=XYZ]
+ *
```

Comment:
does this commit match what you expected @eliorivero ? : https://github.com/Automattic/jetpack/pull/4550/commits/926368d878cfd1ebb99ab020cd01e8acfe983343 ?